### ISSUE TITLE
GPLv2 is not a freeware license

### ISSUE THREAD

**Issue by C0rn3j** [matt-wu/Ext3Fsd#16]
Hi, the readme says that this project is Freeware, when in fact it is Free software(or FOSS) licensed under GPLv2.

https://en.wikipedia.org/wiki/Freeware

https://en.wikipedia.org/wiki/Free_software

**Comment by matt-wu** [comment]
You are right. Freeware is NOT "Free software". Will update the descriptions. Thanks.

**Comment by timofonic** [comment]
@matt-wu Have you fixed it? Please...

**Comment by ngkaho1234** [comment]
The wording is changed in commit https://github.com/matt-wu/Ext3Fsd/commit/1ae63687d39a1ff78ef8edb0117a945072ee1326 .
